pynucastro/networks/rate_collection.py

```diff
@@ -1931,6 +1931,36 @@ def write_network(self, *args, **kwargs):
         assert self._distinguishable_rates(), "ERROR: Rates not uniquely identified by Rate.fname"
         self._write_network(*args, **kwargs)
 
+    def export_as(self, new_type, *args, **kwargs):
+        """Convert the existing network into a different subclass
+
+        Parameters
+        ----------
+        new_type : RateCollection
+            any network type that is based on RateCollection
+        args : Iterable
+            extra arguments to pass into the new class constructor.
+        kwargs : Dict
+            extra keyword arguments to pass into the new class
+            constructor.
+
+        Returns
+        -------
+        RateCollection
+
+        """
+
+        assert issubclass(new_type, RateCollection)
+
+        net = new_type(rates=self.get_rates(), inert_nuclei=self.inert_nuclei,
+                       *args, *kwargs)
+
+        net.symmetric_screening = self.symmetric_screening
+        net.do_screening = self.do_screening
```

Comment:
why not put these in the constructor of `new_type`?

Reply:
okay, I've added some protection about trying to override these, and they are now done via the constructor